src/components/Stories.js

```diff
@@ -203,12 +203,14 @@ const Stories = ({ authorId, followerId }) => {
           <StoriesList stories={stories} isLoading={promiseInProgress} />
         </div>
       )}
-      <Pagination
-        getPage={getPage}
-        storyCount={storyCount}
-        status={currentStateSelected}
-        productQuery={productQuery}
-      />
+      {!checked && (
```

Comment:
Let's make this variable easier to read with a new name from `checked` to something like `isRoadmapView` . This will add clarity that: 1 the variable is a boolean, 2 point directly to the component it affects (roadmap)

Reply:
Done